go-controller/pkg/ovn/base_network_controller_secondary.go

```diff
@@ -327,6 +342,14 @@ func (bsnc *BaseSecondaryNetworkController) addLogicalPortToNetworkForNAD(pod *k
 		bsnc.logicalPortCache.remove(pod, nadName)
 	}
 
+	if shouldCheckLiveMigration {
+		srcPodOps, err := bsnc.createVMSourcePodLSPOpsForLiveMigration(kubevirtLiveMigrationStatus, nadName)
+		if err != nil {
+			return fmt.Errorf("failed to create LSP ops for source pod during Live-migration status: %w", err)
+		}
+		ops = append(ops, srcPodOps...)
```

Comment:
Just pass opts to the function so we can do 
```golang
ops, err := bsnc.createVMSourcePodLSPOpsForLiveMigration(kubevirtLiveMigrationStatus, nadName, ops)
if err != nil {
     return fmt.Errorf("failed to create LSP ops for source pod during Live-migration status: %w", err)
}
```

Reply:
DONE.